zegami_sdk/collection.py

```diff
@@ -1020,6 +1025,35 @@ def __len__(self) -> int:
     def __repr__(self) -> str:
         return "<Collection id={} name={}>".format(self.id, self.name)
 
+    def id_to_class(self, ID):
+        return self.classes[ID - 1]['name']
+
+    def get_annotations_metadata_to_csv(self, outpath='output.csv',
```

Comment:
Personally I'd avoid this method altogether from library as it doesn't bring functionality, if someone wants to save out the data they can use `c.get_annotations_metadata(anno_type).to_csv(path, index=False)`

Reply:
Yeah, I agree in fact. Limiting the surface area of the sdk does have benefits. Perhaps @sofiacbeco could remove in another PR?